playwright/tests/challenges.spec.ts

```diff
@@ -240,3 +255,16 @@ test.describe("Hacker challenge speedrun", () => {
     await expect(completePage.congratulations).toBeVisible();
   });
 });
+
+function getStatedPasswordIn(text: string) {
```

Comment:
This function could be placed in a utils/helper file structure, but that's optional given the overall lack of code separation in this file 🤷

Reply:
Same thought, but if it's fine for you I would leave it as it is